### DISCUSSION TITLE
CopyQ on Windows 10 - press Enter to insert

### DISCUSSION THREAD

**Discussion by francwalter** [hluk/CopyQ#2303] [Q&A]
Hi there.

Is it possible to set up CopyQ on Windows 10 so that I open the list of Clips (I use global shortcut ALT+Y for that) and select an entry (I use ALT+A / ALT+S to navigate down and up, as the arrow keys are too far from the main keyboard) and press ENTER so the selected entry is copied into my document which is open when I pressed ALT+Y ?

I dont find that in the CopyQ shortcuts, I can select an entry and press ESC (to close CopyQ window) and Press CTRL+V to insert, that works, but it is a bit unhandy, could be much faster with ENTER.
The workflow would be (to copy something into my text, what I had copied 3 copies before): ALT+Y, ALT+A, ALT+A, ALT+A, ENTER
(instead of ... ESC, CTRL+V)
It is neither working if I doubleclick an entry to get copied, which would be not that useful, as my mouse is even farer away than my arrow keys 😏 But it shows that the handling of copy seems not normal, I guess.
"Paste and copy next" in global shortcuts is working neither, nothing is pasted into my actual window then. Maybe I misunderstand it.

Momentarily I have in settings > History:

![grafik](https://user-images.githubusercontent.com/927269/226167164-cf64e3d2-0cb0-4a81-92d6-3c5743c6ef34.png)

My global shortcut settings:

![grafik](https://user-images.githubusercontent.com/927269/226168007-cc1b1177-c649-4859-a9ea-2dfb44607acd.png)

My app shortcut settings (the further settings are not relevant I think):

![grafik](https://user-images.githubusercontent.com/927269/226168096-03a9e5b2-f02f-4584-b4f2-eabb7f44797a.png)

Thank for hint.

frank

**Comment by hluk** [comment]
Enter/Return key pastes the selected items by default.

CopyQ sends Shift+Insert to the target application/window. This could be overridden in settings manually.
See also: [FAQ: Why does pasting from CopyQ not work?](https://copyq.readthedocs.io/en/latest/faq.html#why-does-pasting-from-copyq-not-work)
